spirv_reflect.c

```diff
@@ -542,6 +551,52 @@ static SpvReflectTypeDescription* FindType(SpvReflectShaderModule* p_module, uin
   return p_type;
 }
 
+static SpvReflectPrvAccessChain* FindAccessChain(SpvReflectPrvParser* p_parser,
+                                                 uint32_t id) {
+  uint32_t ac_cnt = p_parser->access_chain_count;
+  for (uint32_t i = 0; i < ac_cnt; i++)
```

Comment:
Would you mind wrapping control flow statements in braces? Just to keep uniformity with the existing code? There's a few spots butI'll just leave thie comment on this one.

Reply:
Done.